### PR TITLE
Release 1.28.0

### PR DESCRIPTION
**Pull request by ConnorJC3**
/cc @torredil 
/cc @AndrewSirenko

### CODE REVIEW

**Review by torredil — APPROVED** [review]
/lgtm
/approve
/hold

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+    * In its default behavior, the EBS CSI Driver will attempt to guess the number of reserved volume slots via IMDS metadata (when it is available). Specifying the `--reserved-volume-attachments` CLI option overrides this heuristic value with a user-supplied value.
+    * It is strongly encouraged for users that need to reserve a well-known number of volume slots for non-CSI volumes (such as mounting an extra volume for `/var/lib/docker` data) use this new CLI option to avoid incorrect or incosistent behavior from the heuristic.
+* Report zone via well-known topology key in NodeGetInfo ([#1931](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1931), [@ConnorJC3](https://github.com/ConnorJC3))
+    * A future release of the EBS CSI Driver will migrate the topology key for created volumes from `topology.ebs.csi.aws.com/zone` to the well-known and standard `topology.kubernetes.io/zone`.
+    * After this migration, downgrades of the EBS CSI Driver to versions prior to `v1.28.0` will become impossible in some environments (particularly, environments not running the [AWS CCM](https://github.com/kubernetes/cloud-provider-aws)).
```
Comment (AndrewSirenko) [inline CHANGELOG.md:8]: After this migration -> After this future migration

To give a bit of extra clarity to changelog skimmers.

**Review by AndrewSirenko — APPROVED** [review]

**Pull request merged**